### PR TITLE
use new rv api

### PR DESCRIPTION
**Pull request by taiking**
use new rewarded video api
cf https://developers.google.com/admob/android/rewarded-ads?hl=ja

### CODE REVIEW

File: app/src/main/java/jp/fluct/fluctsdk/exmaple/android/gmamediation/InterstitialActivity.kt [hunk 1]
```diff
@@ -19,7 +19,6 @@ class InterstitialActivity : AppCompatActivity() {
 
         // reigon Optional: 必要に応じて設定してください
         private const val TAG_FOR_CHILD_DIRECTED_TREATMENT = false
-        private const val GENDER = AdRequest.GENDER_UNKNOWN
```
Comment (taiking) [inline app/src/main/java/jp/fluct/fluctsdk/exmaple/android/gmamediation/InterstitialActivity.kt:22]: delete deprecated function 
cf https://developers.google.com/android/reference/com/google/android/gms/ads/AdRequest.Builder#public-adrequest.builder-setgender-int-gender

File: app/src/main/java/jp/fluct/fluctsdk/exmaple/android/gmamediation/InterstitialActivity.kt [hunk 2]
```diff
@@ -44,7 +43,6 @@ class InterstitialActivity : AppCompatActivity() {
 
             ad!!.loadAd(
                 AdRequest.Builder()
-                    .setGender(GENDER)
```
Comment (S64) [inline app/src/main/java/jp/fluct/fluctsdk/exmaple/android/gmamediation/InterstitialActivity.kt:47]: NOTE:
This project was used GMA SDK `v16.x.x` previously, It was updated today: https://github.com/voyagegroup/GMA-Mediation-FluctSDK-Android/pull/8/files#diff-39e7d8c00954e920b98e7636f0ac30b2R49
`AdRequest.Builder::setGender` is deprecated in `v17.x.x`.

**Review by S64 — APPROVED** [review]
💯

**Review by ryokosuge — APPROVED** [review]
👍

**Pull request merged**